### PR TITLE
Recovery the issue 659

### PR DESCRIPTION
**Pull request by Zheaoli**
Recovery the issue 659

### CODE REVIEW

**Comment by Zheaoli** [comment]
#659

**Comment by 38elements** [comment]
If needs of putting instance method in blueprint is nothing, nobody try to put instance method in blueprint.
I think this is unnecessary if nobody try to put instance method in blueprint.

**Comment by Zheaoli** [comment]
Actually,not nobody
In issue 659 , he want to use class and method to make his code structure more clear. Although it's not right , but I can understand it.
So,as what he say, It's necessary to give me some guide/exception for that blueprint handler shouldn't be a instance method

File: sanic/blueprints.py [hunk 1]
```diff
@@ -93,10 +95,16 @@ def route(self, uri, methods=frozenset({'GET'}), host=None,
         :param uri: endpoint at which the route will be accessible.
         :param methods: list of acceptable HTTP methods.
         """
+
```
Comment (r0fls) [inline sanic/blueprints.py]: I'm fine with adding this log mesage, but do the whitespace changes here conform to the rest of the project?

Comment (Zheaoli) [inline sanic/blueprints.py]: oh ,it just format by PyCharm , That`s OK, I will recovery it.

**Comment by 38elements** [comment]
If it is necessary to add exception for instance method,
I think making it possible like #676 is better than adding exception.

**Comment by Zheaoli** [comment]
@38elements 
Seriously , I dont't think it's better than `raise exception` 
here is detail:
If you have N instance method to add in the blueprint , that means you have to make another N `_handler` to decorate it .
If the N is huge , Python have to use a lot of memory to maintain a call stack , I think is a wasting-memory way.
So, I don't think it's right to allow people add instance method in blueprint route.

**Comment by 38elements** [comment]
I think this and #676 are not necessary.
#676 making it possible to put instance method, I think it is better.

**Pull request closed without merging**